deployment/streaming/Makefile

```diff
@@ -25,6 +27,18 @@ BATCH_WORKERS=1
 endif
 endif
 
+ifndef TURBO_BOOST
```

Comment:
🚀

Reply:
I've had a conversation with ops on the client side regarding this over the past week. They are comfortable with us maintaining the higher resource allocations permanently since they plan to do more regular large bulk imports.

As a result, we should just remove the TURBO_BOOST if else but leave the variables in to be configured by the user. In the case of our deployment we want them to be configured at the new higher boosted values:
```
NODE_OPTIONS=--max-old-space-size=7168
DRIVER_MEMORY=8192m
ECS_MEMORY=8
AUGDIFF_ECS_MEMORY=8
```